Autopilot/Src/timeStampingPOGI.cpp

```diff
@@ -7,9 +7,9 @@
 TimeStampingPOGI::TimeStampingPOGI(){
 
     //initializing and starting timer
-    MX_TIM4_Init();
-    HAL_TIM_Base_Init(&htim4);
-    HAL_TIM_Base_Start(&htim4);
+    MX_TIM12_Init();
```

Comment:
You don't need this. If you use CubeMX to configure the ioc this function will be called in `./boardfiles/src/main.c`